### ISSUE TITLE
Create hill-climbing submission for vanilla BO competition

### ISSUE THREAD

**Issue by sgbaird** [AccelerationConsortium/ac-training-lab#428]
Will tipped me off to hill climbing (thanks Willi) which is probably the [closest analogue](https://stats.stackexchange.com/questions/345730/difference-between-hill-climbing-and-gradient-descent) to gradient descent when we don't have gradients.

**Comment by sgbaird** [comment]
Actually, I'm not seeing much by way of [popular implementations](https://github.com/search?q=hill+climbing&type=repositories&s=stars&o=desc), and it might be better to encourage someone who's into hill climbing to do the work for the submission.
